cicd/pipelines/installing-pipelines.adoc

```diff
@@ -17,6 +17,7 @@ This guide walks cluster administrators through the process of installing the {p
 
 * You have installed `oc` CLI.
 * You have installed xref:../../cli_reference/tkn_cli/installing-tkn.adoc#installing-tkn[OpenShift Pipelines (`tkn`) CLI] on your local system.
+* Your cluster has the link:https://docs.openshift.com/container-platform/4.11/post_installation_configuration/cluster-capabilities.html[Marketplace capability] enabled or the Red Hat Operator catalog source configured manually.
```

Comment:
Don't use direct HTML links if the documentation you are pointing to is part of the same repository. Instead, use xref and point towards the correct asciidoc file.

Reply:
For your reference the Serverless PR for the same issue: https://github.com/openshift/openshift-docs/pull/51572/files